docs/common/DocsPageTemplate/index.vue

```diff
@@ -188,4 +238,22 @@
     padding-left: 32px;
   }
 
+  .overlay {
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    z-index: 98;
+    width: 100%;
+    height: 100%;
+    background: rgba(0, 0, 0, 0.7);
+  }
+
+  @media (min-width: 768px) {
```

Comment:
I dont think we need this media query, since we already control the appearance of the overlay with the `v-if` directive